File: program/find-area-of-a-square/find-area-of-a-square.java
import java.util.Scanner;

public class Program
{
    public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
        System.out.print("Enter the side of the square: ");
        int n = sc.nextInt();
        System.out.println(n);
        System.out.print("Area of the square: " + n*n);
	}
}

Output :-
Enter the side of the square: 5
Area of the square: 25

File: program/find-area-of-a-square/find_area_of_a_square.php
<?php
	// Find Area of a Square.
	function find_area_of_a_square($side) {
		if (!is_numeric($side)) {
			return null;
		}
		return $side * $side;
	}
	// Tests
	print_r(find_area_of_a_square(5));      // 25
